pkg/kopia/command/common.go

```diff
@@ -15,16 +15,34 @@
 package command
 
 import (
+	"os"
+
 	"github.com/kanisterio/kanister/pkg/field"
 	"github.com/kanisterio/kanister/pkg/log"
 	"github.com/kanisterio/kanister/pkg/logsafe"
 )
 
+const (
+	// LogLevelVarName is the environment variable that controls Kopia log level.
+	LogLevelVarName = "KOPIA_LOG_LEVEL"
+	// FileLogLevelVarName is the environment variable that controls Kopia file log level.
+	FileLogLevelVarName = "KOPIA_FILE_LOG_LEVEL"
```

Comment:
Just for my understand Eugen, can you please very briefly talk about what is the difference between Log Level and File Log Level?

Reply:
https://kopia.io/docs/advanced/logging/#controlling-log-level

Reply:
Thanks for @hairyhum for link.

Just to have an answer right here:
```
You can control how much data is written to console and log files by using flags:

--log-level - sets log level for console output (defaults to info)
--file-log-level - sets log level for file output (defaults to debug)
```